src/item.h

```diff
@@ -309,6 +309,10 @@ class item : public JsonSerializer, public JsonDeserializer
      */
     bool reduce_charges( long quantity );
     /**
+     * Muzzle flash falloff for guns. 
+     */
+    void muzzle_flash_falloff();
```

Comment:
Should be const. The declaration should probably be where the other gun-related functions are (search for "Gun and gun mod functions").